src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -312,8 +312,8 @@ public boolean isAlive(InetAddressAndPort ep)
             // registration via the metadata log, or a full gossip round). This is perfectly harmless, so no need to log
             // an error in that case.
             ClusterMetadata metadata = ClusterMetadata.current();
-            if (!metadata.directory.allJoinedEndpoints().contains(ep) && !metadata.fullCMSMembers().contains(ep))
-                logger.error("Unknown endpoint: " + ep, new IllegalArgumentException("Unknown endpoint: " + ep));
+//            if (!metadata.directory.allJoinedEndpoints().contains(ep) && !metadata.fullCMSMembers().contains(ep))
```

Comment:
will add back, way too much spam in the test logs making it hard to find *real* issues...